challenges/objects-arrays.js

```diff
@@ -98,28 +237,37 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 */
 
 const lowerCase = [];
-console.log(lowerCase); 
+zooAnimals.map(function(item) {
+  return lowerCase.push(`${item.animal_name.toLowerCase()}`);
```

Comment:
Because `.map()` returns a new array without mutating the original, we can refactor this to:
```const lowerCase = zooAnimals.map(item=> `${item.animal_name.toLowerCase()}`)```